resend.yaml

```diff
@@ -1,93 +1,548 @@
-openapi: 3.1.0
+openapi: 3.0.3
 info:
   title: Resend
   version: 0.1.1
-  description: "Resend is the email platform for developers."
+  description: 'Resend is the email platform for developers.'
 servers:
   - url: https://api.resend.com
-tags:
-  - name: email
-    description: Email operations
 security:
   - bearerAuth: []
+tags:
+  - name: Email
+    description: Start sending emails through the Resend Email API.
+  - name: Domains
+    description: Create and manage domains through the Resend Email API.
+  - name: API Keys
+    description: Create and manage API Keys through the Resend Email API.
 paths:
-  /email:
+  /emails:
```

Comment:
Can you move this endpoint to be the first one since that's what people use the most?

Reply:
This endpoint is already the first one.  Maybe it's something related to the postman itself?